README.md

```diff
@@ -1,5 +1,4 @@
-sbctl - Secure Boot Manager
-===========================
+# sbctl - Secure Boot Manager [![Build Status](https://github.com/Foxboron/sbctl/workflows/build/badge.svg)](https://github.com/Foxboron/sbctl/actions)
```

Comment:
I think it's missing a new line here, isn't it?

Reply:
You would prefer the build status on a new line?

Reply:
I think so, @Foxboron might disagree with me.